app/styles/ui/_file-list.scss

```diff
@@ -15,7 +15,7 @@
   // the list item doesn't have active focus. If it does we
```

Comment:
This comment needs to be updated to indicate that it's now about unselected rows, not focus.

Reply:
:heavy_check_mark: